src/Form/Validator/ErrorElement.php

```diff
@@ -186,13 +188,22 @@ public function addViolation($message, $parameters = [], $value = null)
         }
 
         $subPath = (string) $this->getCurrentPropertyPath();
+        $translationDomain = self::DEFAULT_TRANSLATION_DOMAIN;
+
+        if (\func_num_args() >= 4) {
```

Comment:
Shoule we add a PHPDoc for this @greg0ire ?

Reply:
I think we should 👍